### PR TITLE
Add 3.12.0 changelog

### PR DESCRIPTION
**Pull request by Lex-2008**
@olehermanse / @vpodzime : feel free to take commit and adapt it to your needs in case I stop responding

### CODE REVIEW

**Comment by olehermanse** [comment]
@Lex-2008 Please combine the changelog for beta and final.

**Comment by olehermanse** [comment]
@cf-bottom please trigger a jenkins build

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com/job/pr-pipeline/774/)

https://ci.cfengine.com/job/pr-pipeline/774/

**Comment by olehermanse** [comment]
@Lex-2008 @vpodzime I ran the script again and added the new changes I found, please review my newest commit.

```
python3 misc/changelog-generator/changelog-generator --repo  3.11.0..3.12.x
```

File: ChangeLog [hunk 1]
```diff
@@ -84,6 +116,13 @@
 	- Fix mergedata segfault when called on a non-container (CFE-2704)
 	- Do not segfault if policy_server.dat only contains whitespaces and/or line breaks
 	- Fix segfault on JSON policy files with no bundles and bodies (CFE-2754)
+	- Prevent systemv script from managing processes inside other containers
+	  (ENT-3800)
+	- We must start on line position is 1 instead of 0 to support the @if
```
Comment (vpodzime) [inline ChangeLog]: extra *is* here?

**Pull request merged**